packages/synapse-react-client/src/mocks/msw/handlers/discussionHandlers.ts

```diff
@@ -1,11 +1,40 @@
 import { rest } from 'msw'
-import { FORUM, THREAD } from '../../../utils/APIConstants'
-import { Forum } from '@sage-bionetworks/synapse-types'
+import { FORUM, FORUM_THREAD, THREAD } from '../../../utils/APIConstants'
+import {
+  CreateDiscussionThread,
+  DiscussionFilter,
+  DiscussionThreadBundle,
+  Forum,
+  PaginatedIds,
+  PaginatedResults,
+} from '@sage-bionetworks/synapse-types'
 import { SynapseApiResponse } from '../handlers'
 import {
   mockDiscussionThreadBundles,
   mockForums,
 } from '../../discussion/mock_discussion'
+import { MOCK_USER_ID } from '../../user/mock_user_profile'
+import { uniqueId } from 'lodash-es'
+import mockProject from '../../entity/mockProject'
+
+const forums: Forum[] = [...mockForums]
+
+const threads: DiscussionThreadBundle[] = [...mockDiscussionThreadBundles]
```

Comment:
Configure MSW to use these arrays as an in-memory data store for forums + threads